docs/hacking_howto.rst

```diff
@@ -90,6 +90,8 @@ These are required for the minimum installation:
 * libxslt and headers
 * libjpeg and headers
 * zlib and headers
+* libssl library
+* libmysqlclient library
```

Comment:
I'm not sure what you want me to check. Can you walk me through exactly what I'm checking?